guides/paper-iron-layout.md

```diff
@@ -0,0 +1,320 @@
+---
+title: Responsive layouts with Paper and Iron
+summary: "How to create responsive headers, tabs, side drawers, and 
+icons with Paper and Iron elements."
+tags: ['layout']
+elements: ['paper-header-panel','paper-toolbar','paper-drawer-panel',
+'paper-icon-button','paper-tabs','paper-tab','paper-drawer-panel', 'iron-icons',
+'iron-flex-layout']
+updated: 2015-07-23
+---
+
+## Introduction
+
+This guide teaches you how to use Paper and Iron elements to create a 
+responsive layout.
+
+## Installation
+
+Below is a list of commands for installing all of the elements mentioned
+in this document. You probably
+do not need to install all of these elements. Read the guide and decide
+how you want to implement your layout, and then install only the elements
+that you need.
+
+```bash
+bower install -S PolymerElements/paper-header-panel
+bower install -S PolymerElements/paper-toolbar
+bower install -S PolymerElements/paper-drawer-panel
+bower install -S PolymerElements/paper-icon-button
+bower install -S PolymerElements/paper-tabs
+bower install -S PolymerElements/paper-tab
+bower install -S PolymerElements/paper-drawer-panel
+bower install -S PolymerElements/iron-icons
+bower install -S PolymerElements/iron-flex-layout
+```
+
+We'll assume that you can import these elements from `/bower_components/`.
+
+## Creating a header
+
+This section shows you how to:
+
+* create a standard layout with `paper-header-panel` and `paper-toolbar`,
+  which is probably the most common and easiest layout
+* use a custom element for a header 
+* add icons to a header
+* set the height of a header
+* add tabs to a header
+* modify the disply and behavior of a header
+
+### Creating a header with `paper-toolbar`
+
+The code below uses a `paper-header-panel` as the container of the
+page and a `paper-toolbar` as a header. When a `paper-toolbar` is a 
+child of `paper-header-panel`, the panel automatically displays 
+the toolbar as the header. All other 
+children are placed in the content area.
+
+```hmtl
+...
+<head>
+...
+  <link rel="import" 
+        href="/bower_components/paper-header-panel/paper-header-panel.html">
+  <link rel="import" 
+        href="/bower_components/paper-toolbar/paper-toolbar.html">
+  <link rel="import" 
+        href="/bower_components/iron-flex-layout/iron-flex-layout.html">
+...
+<body class="fullbleed vertical layout">
+  <!-- paper-header-panel must have an explicit height -->
+  <paper-header-panel class="flex">
+    <paper-toolbar>
+      <div>Header</div>
+    </paper-toolbar>
+    <div>Content</div>
+  </paper-header-panel>
+</body>
+...
+```
+
+[Demonstration](assets/header-and-toolbar.html)
+
+`paper-header-panel` **must have an explicit height**. See the list item
+on `flex` below for an explanation of why the code above works.
+
+`fullbleed`, `vertical`, `layout`, and `flex` are part of the 
+`iron-flex-layout` class. We use them in our examples as an easy way 
```

Comment:
maybe "are helper classes included in iron-flex-layout.html"? You could also link directly to classes/iron-flex-layout.html